### PR TITLE
refactor(batch): remove frontend v2 in insert executor and fix some ut

### PR DESCRIPTION
**Pull request by yezizp2012**
## What's changed and what's your intention?

As title.

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:

- Summarize your change (**mandatory**)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Describe clearly one logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)

## Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests

### CODE REVIEW

**Review by BugenZhao — APPROVED** [review]
LGTM! You're too fast! 🤣

File: src/batch/src/executor2/insert.rs [hunk 1]
```diff
@@ -104,16 +94,9 @@ impl InsertExecutor2 {
             ))));
```
Comment (BugenZhao) [inline src/batch/src/executor2/insert.rs:94]:
```rust
let rowid_column = once(Column::from(builder.finish().unwrap()));
```

**Review by liurenjie1024 — APPROVED** [review]
LGTM, seems it has some conflicts with #2548

**Comment by yezizp2012** [comment]
> LGTM, seems it has some conflicts with #2548

Yes, I will resolve the conflicts after #2548 merged.

**Pull request merged**